account_move_tier_validation/tests/test_tier_validation.py

```diff
@@ -88,6 +88,4 @@ def test_03_move_post(self):
         with self.assertRaisesRegex(
             ValidationError, self.env._("You are not allowed to write those fields")
         ):
-            invoice._post()
-        # Calls _post method by passing context skip_validation_check set to True
-        invoice.action_post()
+            invoice.action_post()
```

Comment:
This change looks like it might be related to the test failure. Was it intended?